src/main/java/org/grouplens/grapht/reflect/internal/ClassInstantiator.java

```diff
@@ -49,6 +48,10 @@
     private final Class<?> type;
     private final List<Desire> desires;
     private final Map<Desire, Instantiator> providers;
+    private final InjectionContainer container;
+    private Method[] methods;
+    Annotation[] annotations;
```

Comment:
This field looks unused - it should be deleted (and if it is used, it should probably be a local variable).